static/css/siglens.css

```diff
@@ -2171,7 +2175,7 @@ div#download-info {
 .validateTips.active {
     height: 15px;
     line-height: 12px;
-    font-size: 10px;
+    font-size: 12px;
```

Comment:
increase the font and height slightly more. 
    height: 1em;
    line-height: 1em;
    font-size: 0.8em;